image_provisioner/playbooks/roles/journald-config/tasks/main.yaml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Stop and disable rsyslog
```

Comment:
rsyslog is not installed by default on atomic, so we need to add a condition to disable it only when not atomic. Except that LGTM.